Assets/HoloToolkit/Input/Scripts/Utilities/MixedRealityCameraManager.cs

```diff
@@ -13,19 +13,27 @@ public class MixedRealityCameraManager : Singleton<MixedRealityCameraManager>
     {
         [Tooltip("The near clipping plane distance for an opaque display.")]
         public float NearClipPlane_OpaqueDisplay = 0.3f;
+
         [Tooltip("Values for Camera.clearFlags, determining what to clear when rendering a Camera for an opaque display.")]
         public CameraClearFlags CameraClearFlags_OpaqueDisplay = CameraClearFlags.Skybox;
+
+        [Tooltip("Background color for a transparent display.")]
+        public Color BackgroundColor_OpaqueDisplay = Color.black;
```

Comment:
I don't see this showing up in the inspector:

![image](https://user-images.githubusercontent.com/3580640/30352439-c4560fd0-97d4-11e7-8818-0225877789e2.png)